andela/pulse-mobile-app
# [#156120657] Login Interface

### What does this PR do?
Create the user interface for the Login page

### Description of Task to be completed?
UI implementation for login screen.

### How should this be manually tested?
 - run `react-native run-android` and `react-native run-ios` for android and ios
 -  run `yarn test`. This should run the tests.


### What are the relevant pivotal tracker stories?
[#156120657](https://www.pivotaltracker.com/story/show/156120657)

### Tested on these devices
- Iphone 6
- Iphone 5
- Samsung s8
- Samsung s5

### Screenshots (if appropriate)
![simulator screen shot - iphone 6 - 2018-04-12 at 11 57 23](https://user-images.githubusercontent.com/27197959/38674376-f4eccd80-3e5c-11e8-9036-285414e18b08.png)

![screen shot 2018-04-12 at 11 08 52 am](https://user-images.githubusercontent.com/27197959/38674485-432a1ee4-3e5d-11e8-86b7-9b0181a3e261.png)
